### ISSUE TITLE
Don't highlight apps with available downgrades

### ISSUE THREAD

**Issue by rusty-snake** [NeoApplications/Neo-Store#347]
TL;DR: If an installed app has a newer version than available in repos (i.e. sideloaded dev version for testing or sideloaded from github because repos are outdated), this available downgrade is highlighted. 

I sideload Tor Browser from torproject.org instead of using the Guardian Repo because of faster updates (I get notified about updates via rss feed). When I have the Guardian Repo enabled, I see the update available when browsing through installed apps (and don't even see the app name because of long version numbers). This is also the case for some very old AOSP apps in F-Droid (I would prefer to hide them #121 anyway).
